libstuff/SLog.cpp

```diff
@@ -43,6 +43,7 @@ void SLogStackTrace(int level) {
 // If the param name is not in this whitelist, we will log <REDACTED> in addLogParams.
 static const set<string> PARAMS_WHITELIST = {
     "accountID",
+    "attendees"
```

Comment:
missing comma at the end 😅

Reply:
lool, ouch :(

Reply:
I guess the strings got concatenated and that is why it still compiled

Reply:
@chiragsalian are you preparing a PR to fix it? if not, I can put one up

Reply:
I see Chirag already fixed this here: https://github.com/Expensify/Bedrock/pull/1955, thanks!

Reply:
😄 Great catch